hardwareModules/RoFICoM/software/control_board/src/main.cpp

```diff
@@ -1,7 +1,11 @@
-#include <system/idle.hpp>
-
+#undef NDEBUG
```

Comment:
Jaký je smysl tohoto? Vypadá to jako hack, ale nevím, co se snažíš řešit.

Reply:
To je aby fungoval assert při `RelWithDebInfo`. Zapomněl jsem to smzat.